modules/installation-configuration-parameters.adoc

```diff
@@ -220,6 +220,22 @@ ifdef::osp[]
 |====
 |Parameter|Description|Values
 
+|`compute.platform.openstack.rootVolume.size`
+|For compute machines, the size in gigabytes of the root volume in the machine pool. If you do not set this value, machines use ephemeral storage.
+|Integer, for example `30`.
+
+|`compute.platform.openstack.rootVolume.type`
+|For compute machines, the volume pool from which the volume is created.
```

Comment:
In OpenStack terminology, we're setting the volume `type` - I find it a bit confusing to talk about volume `pools`.

How about: `For compute machines, the type of the root volume.`